src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/_assets.py

```diff
@@ -33,16 +33,27 @@ def _get_asset_references(rop_node: hou.Node) -> AssetReferences:
     return asset_references
 
 
-def _parse_files(node):
-    asset_references = _get_scene_asset_references(node)
+def _parse_files(node: hou.Node):
+    auto_detected_asset_references = _get_scene_asset_references(node)
+    assets = _get_asset_references(node)
+
     for ref in ("input_filenames", "input_directories", "output_directories"):
-        p = node.parm(ref)
-        while p.multiParmInstancesCount():
-            p.removeMultiParmInstance(0)
-        paths = sorted(list(getattr(asset_references, ref)))
-        p.set(len(paths))
-        for i, n in enumerate(p.multiParmInstances()):
-            n.set(paths[i])
+
+        auto_detected_paths = getattr(auto_detected_asset_references, ref)
+        manual_paths = getattr(assets, ref)
```

Comment:
1. No special reason, it can be retrieved 3 times each instead of using getattr likely.
2. I've tested before with no manual assets but I can reconfirm with no scene detected assets
3. Both are Houdini multiparms